File: View/src/app/models/feeder.ts
import {User} from "./user";
import {Tag} from "./tag";
import {Schedule} from "./schedule";

export class Feeder {
  id: number;
  name: string;
  type: string;
  is_Empty: boolean;
  fullness: number;
  is_registered: boolean;
  tags :Tag[];
  user: User;
  schedules: Schedule[];
}

File: Entities/Feeder.cs
namespace FeedYourCat.Entities
{
    public class Feeder
    {
        public int Id { get; set; }
        public int User_Id { get; set; }
        public string Type { get; set; }
        public bool Is_Empty { get; set; }
        public int Fullness { get; set; }
        public bool Is_Registered { get; set; }
        public string Name { get; set; }
    }
}